examples/websites/README.md

```diff
@@ -0,0 +1,53 @@
+#ipfs for websites
+### A short guide to hosting your site on ipfs
+
+Adding your static website to ipfs is quite simple! Simply turn on your daemon:
+```bash
+$ ipfs daemon
+```
+
+And add the directory containing your website:
+```bash
+$ ls mysite
+index.html
+$ ipfs add -r mysite
+added QmbRyftGAtNjXs7hWZUTLyp27qrpgRUXKEwXCVPBueZGek mysite/index.html
+added QmYAMp9ptyn3Bv8ijoYoYAAQHFCfpZ1NKRNedkEWrR9DPH mysite/
+```
+
+The very last hash next to the folder name is the one you want, lets call it
+`$SITE_HASH` for now.  
+
+Now, you can test it out locally by opening `http://localhost:8080/ipfs/$SITE_HASH`
+in your web browser! Next, to view it coming from another ipfs node, you can try
+`http://gateway.ipfs.io/ipfs/$SITE_HASH`. Cool, right?  But those hashes are
+kinda ugly. Lets look at some ways to get rid of them.
```

Comment:
oh missed that somehow... maybe needs to be introduced more. what about:

> Now, you can test it out locally by opening `<URL>` in your web browser! That's neat. 
>
> This is local (note the `localhost` in the URL), but if you're connected to the internet, we can use one of the public ipfs gateways to see it from another IPFS node. Try `<GatewayURL>`, this is an HTTP server that will try to find your website over IPFS, and serve it back to your browser. So the gateway will pull the files out from your node and serve them over HTTP. See the diagram below. Cool, right!?  

BRB making diagrams